Signals/Domain/MissingValuePolicy/SpecificValueMissingValuePolicy.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Domain.Infrastructure;
+using System;
+using System.Collections.Generic;
+
+namespace Domain.MissingValuePolicy
+{
+    public class SpecificValueMissingValuePolicy<T> : MissingValuePolicy
+    {
+        public virtual T Value { get; set; }
+
+        public virtual Quality Quality { get; set; }
+
+        public SpecificValueMissingValuePolicy()
+        {
+            this.Value = default(T);
+            this.Quality = Quality.None;
+        }
+
+        public SpecificValueMissingValuePolicy(T value, Quality quality)
+        {
+            this.Value = value;
+            this.Quality = quality;
+        }
+
+        public override IEnumerable<Datum<S>> FillMissingData<S>(TimeEnumerator timeEnumerator, IEnumerable<Datum<S>> readData)
```

Comment:
Right. My initial idea is to create more than one branch and compare consequences. Indeed, perhaps we should delay merging this branch until this will prove to work out.